client/core/core.go

```diff
@@ -361,14 +364,17 @@ func (dc *dexConnection) parseMatches(msgMatches []*msgjson.Match, checkSigs boo
 		if checkSigs {
 			err := dc.acct.checkSig(sigMsg, msgMatch.Sig)
 			if err != nil {
-				return nil, nil, fmt.Errorf("parseMatches: %v", err)
+				errs = append(errs, err.Error())
+				continue
```

Comment:
Each match has it's own signature, so it seems alright to accept any messages Matches that verify ok.  Although I can't really imagine how it would happen that one fails verification but the others succeed.